tests/plotting/test_plotting.py

```diff
@@ -1254,117 +1272,61 @@ def test_plot_rgb():
+    # p.show()
 
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_cells')
-    p.show()
+    # p = pyvista.Plotter()
+    # p.add_mesh(multicomp_poly, scalars='vector_values_cells')
+    # p.show()
 
     # test component argument
     p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_points', component=0)
+    p.add_mesh(multicomp_poly, scalars='vector_values_points', component=0)
     p.show()
 
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_cells', component=0)
-    p.show()
+    # p = pyvista.Plotter()
+    # p.add_mesh(multicomp_poly, scalars='vector_values_cells', component=0)
+    # p.show()
 
 
-def test_vector_array():
+def test_vector_array(multicomp_poly):
     """Test using vector valued data for image regression."""
-    data = setup_multicomponent_data()
-
     p = pyvista.Plotter(shape=(2, 2))
     p.subplot(0, 0)
-    p.add_mesh(data, scalars="vector_values_points", show_scalar_bar=False)
+    p.add_mesh(multicomp_poly, scalars="vector_values_points", show_scalar_bar=False)
     p.subplot(0, 1)
-    p.add_mesh(data.copy(), scalars="vector_values_points", component=0)
+    p.add_mesh(multicomp_poly.copy(), scalars="vector_values_points", component=0)
     p.subplot(1, 0)
-    p.add_mesh(data.copy(), scalars="vector_values_points", component=1)
+    p.add_mesh(multicomp_poly.copy(), scalars="vector_values_points", component=1)
     p.subplot(1, 1)
-    p.add_mesh(data.copy(), scalars="vector_values_points", component=2)
+    p.add_mesh(multicomp_poly.copy(), scalars="vector_values_points", component=2)
     p.link_views()
     p.show()
 
-    # p.show(before_close_callback=verify_cache_image)
-
-
-def test_vector_plotting_doesnt_modify_data():
-    """Test that the operations in plotting do not modify the data in the mesh."""
-    data = setup_multicomponent_data()
-
-    copy_vector_values_points = data["vector_values_points"].copy()
-    copy_vector_values_cells = data["vector_values_cells"].copy()
-
-    # test that adding a vector with no component parameter to a Plotter instance
-    # does not modify it.
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_points')
-    p.show()
-    assert np.array_equal(data['vector_values_points'], copy_vector_values_points)
-
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_cells')
-    p.show()
-    assert np.array_equal(data['vector_values_cells'], copy_vector_values_cells)
 
-    # test that adding a vector with a component parameter to a Plotter instance
-    # does not modify it.
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_points', component=0)
-    p.show()
-    assert np.array_equal(data['vector_values_points'], copy_vector_values_points)
-
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_cells', component=0)
-    p.show()
-    assert np.array_equal(data['vector_values_cells'], copy_vector_values_cells)
```

Comment:
Was this moved or removed altogether for some reason?

Reply:
Yes, while I was working on the refactor of `setup_multicomponent_data` and forgot about adding the test back in. Fixed in 1e2fe42ac0127f43dc7186daa818c9d7b8c76958.